jepostule/pipeline/models.py

```diff
@@ -187,15 +188,35 @@ def reason_verbose(self):
 
 
 class AnswerEmployerInfo(Answer):
-    employer_name = models.CharField(max_length=128, blank=False)
-    employer_email = models.EmailField(max_length=128, blank=False)
-    employer_phone = models.CharField(max_length=32, blank=False)
-    employer_address = models.CharField(max_length=256, blank=False)
+    employer_name = models.CharField(verbose_name="Prénom et nom du recruteur", max_length=128, blank=True)
+    employer_email = models.EmailField(verbose_name="Email du recruteur", max_length=128, blank=True)
+    employer_phone = models.CharField(verbose_name="Numéro de téléphone", max_length=32, blank=True)
+    employer_address = models.CharField(verbose_name="Adresse de l'entreprise", max_length=256, blank=True)
```

Comment:
TIL What verbose_name does is it adds a label to the side of a database field. It works very much like label for forms in Django. verbose_name is pretty much the equivalent of the label attribute for forms. verbose_name is what is used to give labels to forms.

You would use verbose_name if you don't want the default database field name (in this case, name) to be shown.